src/MultiSelect/index.js

```diff
@@ -161,6 +189,15 @@ class MultiSelect extends Component<Props, State> {
       case 9: // tab
         // Blur input, close dropdown
         return;
+      case 13: // enter
+        if (inputValue !== "") {
+          this.onAddTag({
```

Comment:
For both label and value, let's not modify what the user has typed in. We'll trust that the string they give us is exactly what they'd expect.